Oop-1 Question2 Matrix/Matrix.java

```diff
@@ -0,0 +1,184 @@
+/**
```

Comment:
One thing we must know about exceptional handling if we through some thing we must catch it,otherwise there will be no use of it.Either u must handle it in code or in test class